gradoop-examples/src/main/java/org/gradoop/benchmark/grouping/RollUpExample.java

```diff
@@ -13,16 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.gradoop.examples.rollup;
+package org.gradoop.benchmark.grouping;
```

Comment:
i guess `org.gradoop.examples.grouping` is a better place for this example

Reply:
I actually think a package with name `rollup` within the example package fits best. (Like before the change)